roles/pulp-api/tasks/sso-configuration.yml

```diff
@@ -196,6 +210,7 @@
           keycloak_group_token_claim: "{{ keycloak_group_token_claim | default(omit) }}"
           keycloak_role_token_claim: "{{ keycloak_role_token_claim | default(omit) }}"
           keycloak_host_loopback: "{{ keycloak_host_loopback | default(omit) }}"
+          galaxy_verify_keycloack_ssl_certs: "{{ galaxy_verify_keycloack_ssl_certs | default(omit) }}"
```

Comment:
this is a boolean. Should we go without `" "` here to get a proper value on settings.py?